WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/AddProductToOrderViewModelProtocol.swift

```diff
@@ -0,0 +1,116 @@
+import Yosemite
+
+/// Represents possible statuses for syncing products
+///
+enum SyncStatus {
```

Comment:
This is declared globally for the whole module. Should we add some prefix to keep it specific to single VM?

Reply:
Good point. I renamed this to `AddProductToOrderSyncStatus` in 675552e.